index.html

```diff
@@ -25,28 +25,25 @@ <h3>What We Do</h3>
 		</div>
 	</header>
 
-	<main class="home-main-content">
+	<div class="home-main-content">
 
 		<!-- section 1: -->
 
 		<!-- faq: -->
-<section class="home__faq">
-<div class="home-faq__title"><a href="/about">Frequently Asked Questions</a></div>
-<div class="home-faq__subtitle">Check out <a href="/about/">answers to common questions</a> including:</div>
-
-<ul class="home-faq__questions">
-<li><a href="/about/#what-is-the-scholars-lab">What is the Scholars' Lab?</a></li>
-<li><a href="/about/#how-does-the-lab-fit-into-the-uva-library">How do you fit into the UVA Library?</a></li>
-<li><a href="/about/#how-can-the-scholars-lab-fit-into-my-work-what-do-you-do">What does Scholars' Lab do? How can you fit into my work?</a></li>
-<li><a href="about/#can-you-build-me-a-thing">Can you build me a thing?</a></li>
-<li><a href="about/#can-we-collaborate">Can we collaborate on something?</a></li>
-
-<li><a href="/about/#where-are-you">Where are you?</a></li>
-<li><a href="about/#how-do-i-contact-you">How do I contact you?</a></li>
-</ul>
-
-<div class="home-faq__cta"><!--Find answers to these questions and more:<br/--><a href="/about/">About Scholars' Lab</a></div>
-</section>
```

Comment:
Why was this link removed? I do think it could be reformatted as follows:
```
<div class="home-faq__cta">
<!-- Find answers to these questions and more:<br> -->
<a href="/about/">About Scholars' Lab</a>
</div>	
</section>
```